main.scss

```diff
@@ -8,3 +11,113 @@
 	// Set to silent again to avoid being output twice
 	$o-cookie-message-is-silent: true !global;
 }
+
+/*------------------------------------*\
+		#BLOCK-DESKTOP
+\*------------------------------------*/
+
+.o-cookie-message {
+	position: relative;
+	background-color: #fff1de;
+	border-bottom: 1px solid #d0c6b8;
+	padding: 5px 15px;
+
+	&__container {
+		position: relative;
+		padding-right: 25px;
+	}
+
+	&__close-btn-container {
+		position: absolute;
+		top: 0;
+		right: 0;
+	}
+
+	&__close-btn {
+		position: relative;
+		border-radius: 50%;
+		padding: 0;
+		width: 24px;
+		height: 24px;
+		border: 1px solid getColor('grey-tint5');
+		background: none;
+		cursor: pointer;
+		opacity: 0.75;
+
+		&:before {
+			@include oIconsGetIcon(cross, getColor('grey-tint4'), 12);
+			content: '';
+			position: absolute;
+			top: 5px;
+			left: 5px;
+		}
+
+		&:hover {
+			opacity: 1;
+		}
+	}
+
+	&__title {
+		@include oTypographySansBold(l);
+		margin: 0;
+		color: #4a4a4a;
+		font-size: 18px;
+	}
+
+	&__description {
+		@include oTypographySans(m);
+		margin: 0;
+		font-size: 18px;
+		color: #4a4a4a;
+	}
+
+	&__link {
+		text-decoration: none;
+		color: #27757b;
+		margin-left: 5px;
+	}
+}
+
+/*------------------------------------*\
+		#BLOCK-MODIFIERS
+\*------------------------------------*/
+
+.o-cookie-message--hidden {
+	display: none;
+}
+
+/**
+	* The `--banner-centric` modifier center aligns the content so that it will
+	* align with a 728px width banner that is position beneath the message,
+	* as will be the case on next.ft.com
+	*/
+.o-cookie-message.cookie-message--banner-centric {
+
+	.o-cookie-message__container {
+		max-width: 728px;
+		margin-left: auto;
+		margin-right: auto;
+	}
+}
+
+/*------------------------------------*\
+		#BLOCK-TRANSITIONS
+\*------------------------------------*/
+
+/* always present */
+.t-cookie-message-transition {
```

Comment:
all classes should have the o-cookie-message prefix, so this would be "o-cookie-message__transition" or "o-cookie-message--transition" I don't exactly understand how it's being used